azure-pipelines.yaml

```diff
@@ -7,103 +7,70 @@ pool:
   vmImage: 'Ubuntu 16.04'
 
 variables:
-  GOROOT: '/usr/local/go1.12' # Go installation path
-  GOPATH: '$(System.DefaultWorkingDirectory)/gopath' # Go workspace path
-  GOBIN:  '$(GOPATH)/bin' # Go binaries path
-  MODULE_PATH: '$(GOPATH)/src/$(Build.Repository.Name)' # Path to the module's code
   IMAGE_NAME: 'candidate/k8s/azure-databricks/operator:$(Build.SourceVersion)'
   # ACR_NAME:           - set this value in Azure Devops variables
   # AZURE_SUBSCRIPTION: - set this value in Azure Devops variables
   # DATABRICKS_HOST:    - set this value in Azure Devops variables
   # DATABRICKS_TOKEN:   - set this value in Azure Devops variables
 
-steps:
+jobs:
+- job: Build
+  steps:
+  # Cache the docker image file
+  - task: CacheBeta@0
```

Comment:
What's the difference between `task: Cache@2` and  `CacheBeta@0` ?

Reply:
nice spot, had failed to realise that `Cache` had moved out of beta :)